compiler/gdbmacros/os.gdb

```diff
@@ -34,5 +34,40 @@ end
 
 document os_tasks
 usage: os_tasks
-
 Displays os tasks
+end
+
+define os_callouts
+	printf "Callouts:\n"
+	printf " %8s %10s %10s\n", "tick", "callout", "func"
+	while $c != 0
+		printf " %8d %10p %10p\n", $c->c_ticks, $c, $c->c_ev.ev_cb
```

Comment:
It would be nice to print symbol name for ev_cb in addition to pointer value, but I don't know how and if this is possible.

Reply:
Indeed it would be nice. Alas, AFAIK there's no such gdb printf formatting character, so I'd have to interleave this with 'print', which wants to include additional things in the output.

So I opted for this for now, I left the address as the last thing in format, so we can change it easily later.